react/countdown-karma/webpack.config.js

```diff
@@ -0,0 +1,73 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+const convert = require('koa-connect');
+const history = require('connect-history-api-fallback');
+
+const isTesting = process.env.NODE_ENV === 'testing';
```

Comment:
This probably isn't needed. Should I delete it?

Reply:
Yea